### PR TITLE
Add four collections from alessandro

### PR DESCRIPTION
**Pull request by Schpidi**
@AlessandroScremin as requested, please review

### CODE REVIEW

File: collections/co_3daily_data.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  It is produced primarily by the incomplete combustion or oxidation of carbonaceous materials or substances used for fuel in transportation by internal combustion motor vehicles, in industrial heating and processing, and in the heating of homes and buildings. Other sources of CO production are open burning and aircraft emission.
+  The Copernicus Sentinel-5P CO measurements were first filtered according to the recommendation in the Product Readme file (only data with a qa_value > 0.50 was used). Then the measurements are mapped on a fixed latitude-longitude grid of 4096 x 8192 pixels. The grid is turned into an EPSG:4326 geotiff file using the appropriate color scale, which is again turned into an EPSG:3857 tile map.
+AdditionalInfoExternal:
+  Title: Additional info
+  Path: co_3daily_data/README.MD
+Image: co_3daily_data/co.png
+EDCBrowser: ""
+Resolution: 4.8 x 4.8 Km (across x along track)
+GeographicalCoverage: -180.0, -90.0, 180.0, 90.0
+TemporalAvailability: 2018-04-30 - ongoing
```
Comment (chorng) [inline collections/co_3daily_data.yaml:14]: Maybe add the `TemporalResolution` below `TemporalAvailability`. An example would be [ERA5-Land monthly averaged data](https://github.com/eurodatacube/public-collections/blob/main/collections/reanalysis-era5-land-monthly-means.yaml).

File: collections/co_3daily_data.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+UpdateFrequency: daily
+BandInformation: co
```
Comment (chorng) [inline collections/co_3daily_data.yaml:16]: Please follow the [default structure](https://github.com/eurodatacube/public-collections/blob/main/CONTRIBUTING.md#yaml-file-structure) of `BandInformation`.

File: collections/ocean-net-primary-production.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+Description: |
+  The data comes from the Copernicus Sentinel-5P satellite and uses data from the Copernicus Sentinel-5P satellite and shows the averaged methane concentrations across the globe — using weekly averaged maps. The methane map shown here is measured by the Tropomi instrument on the Sentinel 5 Precursor satellite.
+  The Copernicus Sentinel-5P CH4 measurements were first filtered according to the recommendation in the Product Readme file (only data with a qa_value > 0.50 was used). Then the measurements are mapped on a fixed latitude-longitude grid of 4096 x 8192 pixels. The grid is turned into an EPSG:4326 geotiff file using the appropriate color scale, which is again turned into an EPSG:3857 tile map.
+  Data gaps are visible based on the product quality filtering and the fact that over the sea only measurements for sun-glint situations are being provided.
+AdditionalInfoExternal:
+  Title: Additional info
+  Path: ocean-net-primary-production/README.MD
+Image: ocean-net-primary-production/npp.png
+EDCBrowser: ""
+Resolution: xx x xx Km (across x along track)
```
Comment (chorng) [inline collections/ocean-net-primary-production.yaml:11]: Resolution needs to be filled in.

File: collections/ocean-net-primary-production.yaml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+GeographicalCoverage: -180.0, -90.0, 180.0, 90.0
+TemporalAvailability: 1998 - 2018
+BandInformation: pp
```
Comment (chorng) [inline collections/ocean-net-primary-production.yaml:14]: Please follow the [default structure](https://github.com/eurodatacube/public-collections/blob/main/CONTRIBUTING.md#yaml-file-structure).

File: collections/ocean-net-primary-production.yaml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+      -
+        - -180
+        - -90
+        - 180
+        - 90
+  temporal:
+    interval:
+      -
+        - '1998-01-01T00:00:00Z'
+        - null
```
Comment (chorng) [inline collections/ocean-net-primary-production.yaml:56]: This should not be null if data's `TemporalAvailability` ends on a specific point.

File: collections/ocean-net-primary-production.yaml [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    type: spatial
+    axis: y
+    extent:
+      - -90
+      - 90
+  t:
+    type: temporal
+    extent:
+      - '1998-01-01T00:00:00Z'
+      - null
```
Comment (chorng) [inline collections/ocean-net-primary-production.yaml:74]: Same as above. This should not be null as the data's TemporalAvailability is 1998-2018.

File: collections/ocean-net-primary-production/script.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+  if (sample.dataMask === 0 || sample.pp > 10000){
+    return [0,0,0,0];
+  } else {
+    let rgb = visualizer.process(sample.pp)
+
+
+    return [rgb[0], rgb[1], rgb[2], 1]
+  }
+}
```
Comment (chorng) [inline collections/ocean-net-primary-production/script.js:29]: Would be better to add a new line at the end of file.

File: collections/s5p-ch4-weekly.yaml [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+AdditionalInfoExternal:
+  Title: Additional info
+  Path: s5p-ch4-weekly/README.MD
+Image: s5p-ch4-weekly/ch4.png
+EDCBrowser: ""
+Resolution: 4.8 x 4.8 Km (across x along track)
+GeographicalCoverage: -180.0, -90.0, 180.0, 90.0
+TemporalAvailability: 2021-11-15 - ongoing
+UpdateFrequency: Weekly
+BandInformation: ch4
```
Comment (chorng) [inline collections/s5p-ch4-weekly.yaml:15]: Please follow the [default structure](https://github.com/eurodatacube/public-collections/blob/main/CONTRIBUTING.md#yaml-file-structure).

File: collections/so2_daily_data.yaml [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+AdditionalInfoExternal:
+  Title: Additional info
+  Path: so2_daily_data/README.MD
+Image: so2_daily_data/so2.png
+EDCBrowser: ""
+Resolution: 4.8 x 4.8 Km (across x along track)
+GeographicalCoverage: -180.0, -90.0, 180.0, 90.0
+TemporalAvailability: 2021-09-19 - ongoing
+UpdateFrequency: daily
+BandInformation: so2
```
Comment (chorng) [inline collections/so2_daily_data.yaml:14]: Please follow the [default structure](https://github.com/eurodatacube/public-collections/blob/main/CONTRIBUTING.md#yaml-file-structure).

File: collections/so2_daily_data/script.js [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+function evaluatePixel(sample) {
+
+  var arr = colorBlend(sample.so2,    [0,1,2,3,4, 8 , 12, 16 , 20],    
+  [[248,246,244],[158,97,168],[83,102,182],[60,140,198],[81,172,162], [227,166,37], [235,73,14], [186,0,6], [116,18,13] ]); 
+  if (sample.dataMask==1)  arr.push(255);
+  else arr.push(0);
+  return arr;
+}
```
Comment (chorng) [inline collections/so2_daily_data/script.js:18]: Would be better to add a new line at the end of file.

**Comment by Schpidi** [comment]
@AlessandroScremin could you please take care of these comments from @chorng? Thanks.

**Comment by chorng** [comment]
There are some updates regarding the contribution markdown (see this [pr](https://github.com/eurodatacube/public-collections/pull/226)). @AlessandroScremin Could you please modify the yaml file following the latest structure?

I can also do this in another pull request when I have time.

@Schpidi @AlessandroScremin What is the urgency of this task?

**Comment by chorng** [comment]
@AlessandroScremin Thank you!

**Pull request merged**